crates/runtime/src/dataconnector/s3.rs

```diff
@@ -161,8 +161,14 @@ impl DataConnectorFactory for S3Factory {
                 validator.validate(&mut params).await?;
             }
 
-            // Initialize the AWS SDK and make it available.
-            let _ = object_store_aws_sdk::initialize_sdk_config().await;
+            // `initialize_sdk_config` emits a warning if the credentials provider cannot be initialized
```

Comment:
We shouldn't do this - we can suppress the warning message from loading the SDK if it can't be loaded instead.